recipes/braker2/path.patch

```diff
@@ -1,62 +1,111 @@
---- scripts/align2hints.pl	2018-09-10 08:36:32.000000000 -0500
-+++ scripts/align2hints.pl.new	2019-04-13 00:45:20.199730500 -0500
+diff --git a/scripts/align2hints.pl b/scripts/align2hints.pl
+index aeacb40..b64ae25 100755
+--- a/scripts/align2hints.pl
++++ b/scripts/align2hints.pl
 @@ -1,4 +1,4 @@
--#!/usr/bin/perl
+-#!/usr/bin/env perl
```

Comment:
Here you haven't used the miniconda one?

Reply:
I have, look at the next line down.